src/parser/csql_grammar.y

```diff
@@ -27533,3 +27568,48 @@ pt_jt_append_column_or_nested_node (PT_NODE * jt_node, PT_NODE * jt_col_or_neste
     }
 }
 
+static PT_NODE *
+pt_make_paren_expr_list (PT_NODE * exp)
```

Comment:
naming: this function name should start with "pt_make" or "pt_create"?

Reply:
yes. I changed name to "pt_create..."